meetballs/settings.py

```diff
@@ -37,7 +37,11 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'meet_balls_app.apps.MeetBallsAppConfig'
+    'meet_balls_app.apps.MeetBallsAppConfig',
+    'court.apps.CourtConfig',
+    'CourtBallGame.apps.CourtBallGameConfig',
+    'meet_balls_app.apps.MeetBallsAppConfig',
```

Comment:
you have duplicate field 'meet_balls_app.apps.MeetBallsAppConfig',
and 
'court.apps.CourtConfig',

Reply:
Done